compiler/damlc/lib/DA/Cli/Damlc.hs

```diff
@@ -726,8 +762,7 @@ execRepl dars importPkgs mbLedgerConfig mbAuthToken mbAppId mbSslConf mbMaxInbou
             -- We change directory so make this absolute
             dars <- mapM makeAbsolute dars
             opts <- pure opts
-                { optDlintUsage = DlintDisabled
```

Comment:
now that `cmdRepl` forbids any linter flags, we don't need to hardcode it off here.